Rocket.ChatTests/CustomFields/TextFieldSpec.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TextFieldSpec.swift
+//  Rocket.ChatTests
+//
+//  Created by Vadym Brusko on 10/17/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import XCTest
+import SwiftyJSON
+
+@testable import Rocket_Chat
+
+class TextFieldSpec: XCTestCase {
+    func testMap() {
+        // arrange
+        let textField = TextField()
+        let minLength = -5
+        let maxLength = 5
+        let json = JSON(["minLength": minLength,
+                         "maxLength": maxLength])
```

Comment:
Can you just use our code style to create the dictionary? It will become something like this:

```swift
let json = JSON([
    "minLength": minLength,
    "maxLength": maxLength
])
```